src/vma/sock/sockinfo_udp.cpp

```diff
@@ -767,14 +767,12 @@ int sockinfo_udp::setsockopt(int __level, int __optname, __const void *__optval,
 	if (unlikely(m_b_closed) || unlikely(g_b_exit))
 		return orig_os_api.setsockopt(m_fd, __level, __optname, __optval, __optlen);
 
-#ifdef DEFINED_SOCKETXTREME
 	/* Process VMA specific options only at the moment
 	 * VMA option does not require additional processing after return
 	 */
 	if (0 == sockinfo::setsockopt(__level, __optname, __optval, __optlen)) {
```

Comment:
Can we also remove the comment if line 771? It's very confusing,
It exsit in 4 diffrent locations (tcp/udp setsockopt/getsockopt